DigitalLearningSolutions.Data/Services/DelegateUploadFileService.cs

```diff
@@ -187,8 +187,8 @@ private void UpdateDelegate(DelegateTableRow delegateRow, DelegateUser delegateU
         private void RegisterDelegate(DelegateTableRow delegateRow, DateTime? welcomeEmailDate, int centreId)
         {
             var model = new DelegateRegistrationModel(delegateRow, centreId, welcomeEmailDate);
-            var status = registrationDataService.RegisterDelegateByCentre(model);
-            switch (status)
+            var statusOrCandidateNumber = registrationDataService.RegisterDelegateByCentre(model);
+            switch (statusOrCandidateNumber)

```

Comment:
Ah, good point - could we just tweak this to errorCodeOrCandidateNumber? That makes it a bit clearer that it's not a candidate number when an error's occurred. Currently it's not so obvious when it's a status and when it's a candidate number. No need for re-review. Unless it demands approval before letting you merge.

Reply:
Changed to errorCodeOrCandidateNumber. No need for re-review, so I will merge it once the checks pass.